### PR TITLE
[RU] tranlsation in files 01-1.md and README-RU.md

### PR DESCRIPTION
**Pull request by MrPark97**
Full tranlation of REDME and 01-1.md also config changed. Wiki has already created by me. All terms translated in Russian Wiki page.

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -525,3 +525,14 @@ fr:
...
+
+#################################### Russian ####################################
+ru:
+  title: 'Глубокое обучение'
+  chapters:
+    - path: ru/week01/01.md
+      sections:
+        - path: ru/week01/01-1.md
+        - path: ru/week01/01-2.md
+        - path: ru/week01/01-3.md
```
Comment (xcastilla) [inline docs/_config.yml:538]:
```suggestion
```
There is no `ru/week01/01-2.md` or `ru/week01/01-3.md` in the current tree nor in this PR. Let's add those lines when we have a PR for them.

**Comment by xcastilla** [comment]
I can't comment on the translation itself, so let's wait for someone who can. For the rest, it looks good to me!

File: docs/_config.yml (hunk 1 above)
Comment (MrPark97) [reply docs/_config.yml]: Lines have added

**Comment by MrPark97** [comment]
Ok, so I will translate other files, and will open another PR with new files

**Comment by Atcold** [comment]
No need to keep opening and closing PRs.
You can use a new branch per file *or* keep adding files to the same branch.
These PRs will automatically get updated with what you do.

**Pull request closed without merging**